ui-src/src/event-listeners/index.js

```diff
@@ -38,6 +46,10 @@ export default function setupEventListeners(store, canvas) {
       case "KeyG":
         store.dispatch(unsetGKeyDown())
         break
+      case "ShiftLeft":
+      case "ShiftRight":
+        store.dispatch(unsetShiftKeyDown())
```

Comment:
I was wondering how to make cases with two matches. This is sweet!

Reply:
Yeah, so until you put a 'break' or 'return' statement, it will still execute the code 'underneath' that follows it

Reply:
huh, that's a deceivingly simple implementation of switch/case

Reply:
almost reminds me of using GOTO..